### DISCUSSION TITLE
MaxCDN no longer working, clicking on emojis to get SVG broken

### DISCUSSION THREAD

**Discussion by phil-hudson** [ShahriarKh/twemoji-cheatsheet#14] [Bugs] [answered]
Per, https://github.com/twitter/twemoji/issues/580 - can we swap out the emojiURL to Cloudflare?

To reproduce: just click on an emoji to get to the svg file in prod, you'll see the request fail

**Comment by ShahriarKh** [comment, marked as answer]
Thank you for reporting. I switched to jsdelivr.

**Reply by phil-hudson** [reply]
Thank you! :)
